web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.ts

```diff
@@ -32,6 +32,11 @@ const settings = {
       }
     ]
   },
+  SourceFormatDelimiter: {
+    label: 'Source Format Delimiter',
+    description: 'The delimiter character used in the delimited file. Accepted values are comma(,), tab(" "), Default: (,)',
```

Comment:
Change the description:
'The delimiter character used in the delimited file. Accepted values are comma(,) and tab(" "). Default: comma (,)'